interpreter/CMakeLists.txt

```diff
@@ -86,6 +86,13 @@ else()
   set(ROOT_CLING_TARGET "all")
 endif()
 
+if(GENERATOR_IS_MULTI_CONFIG AND NOT LLVM_BUILD_TYPE MATCHES "Debug")
```

Comment:
You may want to consider matching "RelWithDebInfo" as well, and making it case-insensitive.
A suggestion to let you better control `LLVM_BUILD_TYPE` is:
```cmake
if(GENERATOR_IS_MULTI_CONFIG)
  string(TOUPPER ${LLVM_BUILD_TYPE} LLVM_BUILD_TYPE)
  set(LLVM_C_FLAGS ${CMAKE_C_FLAGS_${LLVM_BUILD_TYPE}})
  set(LLVM_CXX_FLAGS ${CMAKE_CXX_FLAGS_${LLVM_BUILD_TYPE}})
  foreach(BUILD_TYPE DEBUG RELEASE RELWITHDEBINFO MINSIZEREL)
    set(CMAKE_C_FLAGS_${BUILD_TYPE} ${LLVM_C_FLAGS})
    set(CMAKE_CXX_FLAGS_${BUILD_TYPE} ${LLVM_CXX_FLAGS})
  endforeach()
endif()
```